numba/cuda/tests/cudadrv/test_cuda_driver.py

```diff
@@ -113,6 +113,15 @@ def test_cuda_driver_stream(self):
         for i, v in enumerate(array):
             self.assertEqual(i, v)
 
+    def test_cuda_driver_occupancy(self):
+        module = self.context.create_module_ptx(self.ptx)
+        function = module.get_function('_Z10helloworldPi')
+
+        value = self.context.get_active_blocks_per_multiprocessor(function, 128, 128)
+        print('active blocks:', value)
```

Comment:
I think that although there's not a great deal we can do to test the occupancy calculator, you could check that the results of `get_active_blocks_per_multiprocessor` and `get_max_potential_block_size` are nonzero integers.

It would be good to avoid making the test suite noisy (or, noisier), so could you remove the prints? In practice, I think that the output won't be inspected, and it's hard to know what to expect the value to be when inspecting.